internal/payloads/error_payload.go

```diff
@@ -54,7 +55,8 @@ func newResponse(ctx context.Context, status int, userMsg string, err error) *Re
 		strError = err.Error()
 	}
 	if userMsg == "" {
-		userMsg = err.Error()
+		// take only part up to the first colon to avoid unique ids (UUIDs, database IDs etc)
+		userMsg = strings.SplitN(err.Error(), ":", 2)[0]
```

Comment:
There is this change.

When we get an error in service package and return it to the user as a HTTP payload, previously, we used the whole error message as a log message. This is an attempt to split these errors:

something happen: something: blah: aws error: request = UUID XXXXXXXX

to just

something happen

This will only works for errors with colons which are most of them in our app. Alternatively, we can give this a log line something like "Returned an error as payload" but that is quite lame.